### PR TITLE
Basic structure rockets

### LINKED ISSUE #21: [1pt] Create basic structure for Rockets - Setup
- Create a route and a view component. Use `<NavLink />` for the page navigation links and style active class to indicate which section/page the user is currently on (underline active navigation link).
- Create a directory for all Redux state slice files.

### PR DESCRIPTION
**Pull request by Hamzaoutdoors**
# Create basic structure for Rockets - Setup 
Hey @notlfish , in this pull request, I've been working on creating the basic structure for Missions.

 - Create a route and a view component. Use <NavLink /> for the page navigation links and style active class to indicate which section/page the user is currently on (underline active navigation link).
 - Create a directory for all Redux state slice files.
 - Add some animation for the ``logo`` to practice using ``Framer-motion``

### CODE REVIEW

File: src/components/navbar/Logo.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    <motion.div
+      variants={logoContainer}
+      initial="initial"
+      animate="animate"
+      className={styles.textContainer}
+    >
+      {animateLogo()}
+    </motion.div>
+  );
+};
```
Comment (notlfish) [inline src/components/navbar/Logo.js:49]: Nice job going for the extra mile and giving some nice visual effects to the page 👏🏾👏🏾👏🏾👏🏾👏🏾

File: src/components/navbar/logoStyle.module.scss [hunk 2]
```diff
@@ -0,0 +1,7 @@
+.textContainer {
+  display: flex;
+  flex-direction: row;
+  padding: 0 1rem;
+  gap: 3px !important;
```
Comment (notlfish) [inline src/components/navbar/logoStyle.module.scss:5]: We could eventually have problems with the code reviewers for using !important. Please, keep them to a minimum.

**Review by notlfish — COMMENTED** [review]
# Approved
Nice job with this milestone! You can now merge it!

**Review by notlfish — APPROVED** [review]
Sorry, I didn't mark the previous review as approved.

File: src/components/navbar/logoStyle.module.scss (hunk 2 above)
Comment (Hamzaoutdoors) [reply src/components/navbar/logoStyle.module.scss:5]: Yes, you are right .. I will remove it 🚀

**Pull request merged**